content/soarian/dstu2.md

```diff
@@ -0,0 +1,69 @@
+
+![Soarian API and Application interaction](/images/soarian-api-application-interaction.jpg)
+
+Once the IdP, Application, and API are registered, API is enabled and the user-patient-proxy identity and authorization is established, the patient/proxy Application user can use Application capabilities to query the API for:
+ * One or more C-CDA SOEN (consolidated clinical documentation architecture summary of episode note) containing an authorized patient's health information 
+ * Each category of health data such as a medication list, allergy list, problem list, diagnostic test results, or other specified information.  
+
+The Application is responsible for determining which of and how the API capabilities are exposed to the user. For each capability the Application offers the user, the FHIR resource pages define those API capabilities and, within each, which capabilities the Application must/shall support to be compliant with the API.
+
+## Special information regarding date parameters
+
+* All date formats are ISO 8601 formatted: YYYYMMDD.
+
+* Date is assumed formatted properly and always evaluated left to right: YYYYMMDD.  Note specifically that if 20152101 is used, MM is 21 or 12+9 months and the API will resove 20152101 to 20160901.
+
+* While DSTU2 uses a letter syntax for date (e.g., ge, le, eq), the API utilizes symbols (e.g., >=, <=, =).  Syntax for optional date parameter entry includes:
+
+* &date=\[YYYYMMDD].  Exactly and inclusive of the specified date.  Note that a date passed without a comparator (e.g., <) is interpreted as an exact date per FHIR DSTU 2.
+
+* &date=\[< YYYYMMDD].  Less than the date, not inclusive of the date.
+
+* &date=\[<= YYYYMMDD].  Less than or equal to the date, inclusive of the date.
+
+* &date=\[> YYYYMMDD].  Greater than the date, not inclusive of the date.
+
+* &date=\[>= YYYYMMDD].  Greater than or equal to the date, inclusive of the date.
+
+For the following range descriptions, entered sequence of high and low dates is immaterial, the API evalutes the symbols regardless of entry sequence.
+
+* &date=\[> YYYYMMDD]&date=\[< YYYYMMDD].  Between date and date, not inclusive of either.
+
+* &date=\[>= YYYYMMDD]&date=\[<= YYYYMMDD].  Between date and date, inclusive of both.
+
+* &date=\[> YYYYMMDD]&date=\[<= YYYYMMDD].  Between date and date, inclusive of the high date, exclusive of the low date.
+
+* &date=\[>= YYYYMMDD]&date=\[< YYYYMMDD].  Between date and date, exclusive of the high date, inclusive of the low date.
+
+* The API accepts single, contiguous, non-zero, positive date ranges.  An entry of >high date and < low date is an invalid date range; for example, &date=\[>20160101]&date=\[<= 20150601], is two date ranges of less than June 2015 and greater than January 2016.  The date range entered will be returned in the response for user re-evaluation information.
+
+## Common Errors
+
+- [Common errors][COMMON-ERRORS]
```

Comment:
Is it really useful to have the common errors table on a separate page?  Is it linked to from anywhere else?